opencodelists/models.py

```diff
@@ -32,11 +34,12 @@ def create_user(self, username, email, password, **extra_fields):
             )
 
         email = self.normalize_email(email)
-        user = self.model(
-            username=username, email=email, organisation=org, **extra_fields
-        )
+        user = self.model(username=username, email=email, **extra_fields)
         user.set_password(password)
         user.save()
+        user.memberships.create(
+            organisation=org, date_joined=datetime.today(), is_admin=False
+        )
```

Comment:
Does `User.objects.create_user(...)` fail without this change?

Users no longer need to belong to an organisation, so I think it'd be better to remove any organisation-handling code from this function.

Reply:
Yes, because it's trying to create a user with an organisation kwarg that doesn't exist anymore.  I made it do the old thing of creating a default org, but I could just make it skip the organisation bit, if it's not necessary.  (I only discovered it because I tried to do `./manage.py createsuperuser` for my local env, which failed)

Reply:
> I could just make it skip the organisation bit, if it's not necessary

I'd ditch it.